ct-aa/dotTheory---TypeScript
# JS-971: Added the sample project using webpack with typescript

# Related Tickets
[JS-971](https://codeandtheory.atlassian.net/browse/JS-971)

# Description
- Added the sample project using webpack with typescript.

# Validation
- Open the terminal.
- Run "cd webpack-with-typescript" to change the directory.
- Run "npm build".
- Run "tsc -w" to watch the TypeScript files.
- Run `npm run start`.
- Go to the browser. The Drag and Drop app with webpack should display and function properly.

<img width="1752" alt="Screen Shot 2024-01-16 at 4 35 53 PM" src="https://github.com/ct-aa/dotTheory---TypeScript/assets/93116252/144db2ac-df96-4e2a-b813-fa1a1f552cff">

# Risks
N/A
